test/simplifyExpression/multiplyFractionsSearch/multiplyFractionsSearch.test.js

```diff
@@ -8,9 +8,13 @@ function testMultiplyFractionsSearch(exprString, outputStr) {
 
 describe('multiplyFractions', function () {
```

Comment:
well, sometimes you can cancel without having to multiply first

funny, I see it more like the only case for multiplying fractions is to cancel :p especially since that's the step that users see